test/cases/parsing/evaluate/index.js

```diff
@@ -20,32 +24,46 @@ it("should evaluate logical expression", function() {
 	expect(value5).toBe("value5");
 	expect(value6).toBe("value6");
 	expect(value7).toBe(false);
+	expect(value8).toBe(true);
+	expect(value9).toBe(false);
+	expect(value91).toBe(false);
+	expect(value92).toBe(false);
+
+	if (!process.version.startsWith("v14")) return;
+
+	const value10 = "" ?? require("fail");
```

Comment:
since we render `"" ?? false` need to find a way to load this tests in Node.js < 14.. Maybe use babel before load..